src/phantom/base/typography/_bundle.scss

```diff
@@ -2,3 +2,15 @@
 @import "links_config";
 @import "lists_config";
 @import "paragraphs_config";
+
+@font-face {
+  font-family: "Calibre Light";
+  src: url('https://casper.com/static-assets/nightshade-core/src/typography/fonts/CalibreWeb-Light.woff2') format('woff2'),
```

Comment:
@jpsilvashy yes, i agree with you. i like the new bucket route and that's a clear path for us. when we reference the files in sass file, should we just use the aws url or cdn url?